### PR TITLE
Add tariff management service tests

### PR DESCRIPTION
**Pull request by aianushevich**
- Add tariff management service tests
- Add RunTestsSuite class for junit runs

### CODE REVIEW

File: rest-assured-tests/src/test/java/tariffManagementService/TariffObject.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package tariffManagementService;
+
+import com.google.gson.Gson;
+import io.restassured.response.Response;
+
+class TariffObject {
+    int id;
```
Comment (AfanasievYuriy) [inline rest-assured-tests/src/test/java/tariffManagementService/TariffObject.java]: class can be public I guess
private on fields
@Getter/Setter on class

File: rest-assured-tests/src/test/java/tariffManagementService/TariffTests.java [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    private final static String TARIFF_NAME_TO_DELETE = "NewTestTariffToDelete";
+    private final static String VALID_TARIFF_NAME = "NewTestTariff";
+    private final static String INVALID_TARIFF_NAME = "TariffWillNeverExist";
+    private final static String TARIFF_NAME_TO_DUPLICATE = "DuplicatedTariffName";
+    private final static float VALID_TARIFF_PRICE = 4.67f;
+    private final static int VALID_TARIFF_WASHING_TIME = 12;
+    private final static float INVALID_TARIFF_PRICE = -1.24f;
+    private final static int INVALID_TARIFF_WASHING_TIME = -7;
+    private final static int DELTA = 0;
+    private final static String INVALID_PARAMETER_MESSAGE = "Tariff not created, price and washing time should be a non negative";
```
Comment (AfanasievYuriy) [inline rest-assured-tests/src/test/java/tariffManagementService/TariffTests.java:26]: Is this exact text from tariff-service's responce?

**Review by AfanasievYuriy — CHANGES_REQUESTED** [review]

File: rest-assured-tests/src/test/java/tariffManagementService/TariffTests.java (hunk 2 above)
Comment (aianushevich) [reply rest-assured-tests/src/test/java/tariffManagementService/TariffTests.java:26]: yes, it is

File: rest-assured-tests/src/test/java/tariffManagementService/TariffObject.java (hunk 1 above)
Comment (aianushevich) [reply rest-assured-tests/src/test/java/tariffManagementService/TariffObject.java]: done

File: rest-assured-tests/src/test/java/commonTestClasses/FailedResponse.java [hunk 3]
```diff
@@ -4,9 +4,21 @@
...
 
 public class FailedResponse {
-    public int status;
-    public String error;
-    public String message;
+    private int status;
+    private String error;
+    private String message;
+
+    public int getStatus() {
```
Comment (AfanasievYuriy) [inline rest-assured-tests/src/test/java/commonTestClasses/FailedResponse.java]: Use lombok https://projectlombok.org/

**Review by AfanasievYuriy — CHANGES_REQUESTED** [review]

File: rest-assured-tests/src/test/java/commonTestClasses/FailedResponse.java (hunk 3 above)
Comment (aianushevich) [reply rest-assured-tests/src/test/java/commonTestClasses/FailedResponse.java]: done

File: rest-assured-tests/src/test/java/commonTestClasses/FailedResponse.java [hunk 4]
```diff
@@ -2,23 +2,12 @@
...
-    public String getError() {
-        return error;
-    }
-
-    public String getMessage() {
-        return message;
-    }
+    private @Getter int status;
+    private @Getter String error;
+    private @Getter String message;
```
Comment (AfanasievYuriy) [inline rest-assured-tests/src/test/java/commonTestClasses/FailedResponse.java]: you can set @Getter annotation on class
(by the way, you can always check out code in service itself to get examples)

**Review by AfanasievYuriy — CHANGES_REQUESTED** [review]

File: rest-assured-tests/src/test/java/commonTestClasses/FailedResponse.java (hunk 4 above)
Comment (aianushevich) [reply rest-assured-tests/src/test/java/commonTestClasses/FailedResponse.java]: ok, tnx. I've updated it.

**Review by AfanasievYuriy — APPROVED** [review]

**Pull request merged**